resolwe/flow/models/storage.py

```diff
@@ -5,6 +5,31 @@
 from django.db import models
 
 from .base import BaseModel
+from .functions import JsonGetPath
+
+
+class StorageManager(models.Manager):
+    """Manager for Storage objects."""
+
+    def with_json_path(self, path, field='result'):
+        """Annotate Storage objects with a specific JSON path.
+
+        :param path: Path to get inside the stored object, which can be
+            either a list of path components or a comma-separated
```

Comment:
Dot-separated string should be also supported, to make it similar to `dict_dot`.

Reply:
We could also just drop everything and support just lists/tuples of path components. I don't actually like these different formats here and it would also make reasoning about paths easier (e.g. as mentioned in the comment above).